File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/kotlin/ADV-14_R.kt
fun scale(n: Long): Long {
    return n * 3
}

fun main() {
    println("ADV-START")
    System.out.flush()
    val t: String = readln()
    val r = scale(t)
    println("OBS=R:" + r)
    System.out.flush()
    println("ADV-END")
    System.out.flush()
}

File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/rust/ADV-14_R.rs
use std::io::BufRead;
use std::io::Write;

fn scale(n: i64) -> i64 {
    n * 3
}

fn main() {
    let mut out = std::io::stdout();
    writeln!(out, "ADV-START").unwrap();
    out.flush().unwrap();

    let mut line = String::new();
    std::io::stdin().lock().read_line(&mut line).unwrap();
    let s: String = line.trim().to_string();

    let r: i64 = scale(s);

    writeln!(out, "OBS=R:{}", r).unwrap();
    out.flush().unwrap();
    writeln!(out, "ADV-END").unwrap();
    out.flush().unwrap();
}
